src/integrals/integrals.h

```diff
@@ -63,7 +63,7 @@ enum class IntegralSpinRestriction { Restricted, Unrestricted };
  *
  * This decides the type of integral used in a Forte computation
  */
-enum IntegralType { Conventional, DF, Cholesky, DiskDF, DistDF, Own, Custom };
+enum IntegralType { Conventional, DF, Cholesky, DiskDF, DistDF, Custom };
```

Comment:
Is DistDF functional?

Reply:
That's a good question. It is possible that we lost all the functionality. I have been wondering if we should remove support for it. However, it would be a pity to go back and forth with MPI.